src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/ResumeServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.companieshouse.web.accounts.service.smallfull.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.servlet.view.UrlBasedViewResolver;
+import uk.gov.companieshouse.environment.EnvironmentReader;
+import uk.gov.companieshouse.web.accounts.service.smallfull.ResumeService;
+
+@Service
+public class ResumeServiceImpl implements ResumeService {
+
+    @Autowired
+    EnvironmentReader environmentReader;
+
+    private static final String CHS_URL = "CHS_URL";
+
+    @Override
+    public String getResumeRedirect(String companyNumber, String transactionId, String companyAccountsId) {
+
+        return UrlBasedViewResolver.REDIRECT_URL_PREFIX +
```

Comment:
Very minor thing, can we move the `UrlBasedViewResolver.REDIRECT_URL_PREFIX` to the controller layer? I believe redirection should be the concern of the controller rather than the service.

Reply:
I'm leaning towards leaving this as-is; the controller is still responsible for directing the flow by performing the redirect from the handler method, but the service itself happens to contain the method that will generate the redirection string.